mappings/net/minecraft/client/render/model/BakedModel.mapping

```diff
@@ -13,5 +13,8 @@ CLASS net/minecraft/class_1087 net/minecraft/client/render/model/BakedModel
 	METHOD method_4709 getTransformation ()Lnet/minecraft/class_809;
 	METHOD method_4710 getOverrides ()Lnet/minecraft/class_806;
 	METHOD method_4711 getSprite ()Lnet/minecraft/class_1058;
```

Comment:
```suggestion
	METHOD method_4711 getModelIcon ()Lnet/minecraft/class_1058;
```
perhaps?

Reply:
`MinecraftClient.checkGameData` for context

However, looking at `InGameOverlayRenderer#renderOverlays` and `InGameHud#renderPortalOverlay`, the sprite is used in full than just for an icon/particle? Also fluid renderer seems to use the sprite wholesale too

Reply:
It's like a "representation" of the model, the most important texture, icon just felt most fitting.

Reply:
Good idea! But if you do rename, remember to update the associated `getSprite` methods in `BlockModels` and `ItemModels` to `getIcon` as well (I think `getIcon` is enough, no need for `Model` in name)